apps/trading/components/market-selector/product-selector.tsx

```diff
@@ -11,11 +11,12 @@ export const Product = {
   Perpetual: 'Perpetual',
 } as const;
 
-export type ProductType = keyof typeof Product;
+export type ProductType = keyof typeof Product | 'All';
```

Comment:
Why create a union here? the const above is only for ui purposes so I'd just add a key to the ProductType const

Reply:
ok, there are a little bit misleading comment: `// Make sure these match the available __typename properties on product` 
Sure, gonna improve it

Reply:
done